test/TEST-16-ISSUE-518/avocado.conf

```diff
@@ -0,0 +1,2 @@
+[datadir.paths]
+logs_dir = /tmp/job-results
```

Comment:
> It would be nice to agree (or at least advise to people) on the test framework if they need to write "integration test" for systemd

Well, `test/networkd-test.py`, `test/sysv-generator-test.py` use `unittest`

> there are no other "issue specific" tests anywhere else in systemd source tree, except test/

Hm, my bad 09f6f45a29d869

Anyway, https://github.com/systemd/systemd/blob/master/test/README.testsuite mentions "the extended testsuite". I think we shouldn't mix that with other frameworks.

> Yes it will use systemd from the system but I don't think it is a job of the test case to prepare test environment

Maybe, but we should place such tests into `test/TEST-*`.